README.md

```diff
@@ -106,7 +105,7 @@ A list of tested compilers which are currently not working and the respective is
 
 Name | Version | Platform | Architecture | Status
 --- | --- | --- | --- | ---
-GCC Fortran | 7.4 | Windows 10 | i686 | [#296](https://github.com/fortran-lang/stdlib/issues/296)
+GCC Fortran | <9 | any | any | [#430](https://github.com/fortran-lang/stdlib/pull/430)
```

Comment:
I think it would be good to keep the reference to #298, in case that someone is still using GCC Fortran 7.4 on Windows.

Reply:
```suggestion
GCC Fortran | <9 | any | any |  [#296](https://github.com/fortran-lang/stdlib/issues/296), [#430](https://github.com/fortran-lang/stdlib/pull/430)
```